src/components/cc-logs-instances/cc-logs-instances.stories.js

```diff
@@ -22,6 +23,7 @@ function getDeploymentDate (sinceDay) {
 }
 
 export default {
+  tags: ['autodocs'],
```

Comment:
About your first question, as discussed in sync it's because I forgot to update newly added files from the last rebase.

About your second, you are right, nice catch :+1: